src/app/portlets/dot-edit-content/dot-edit-content.component.spec.ts

```diff
@@ -313,4 +354,21 @@ describe('DotEditContentComponent', () => {
         expect(workflowService.getPageWorkflows).toHaveBeenCalledWith('123');
         expect(dotEditContentHtmlService.renderPage).toHaveBeenCalledWith('<html></html>', component.iframe);
     });
+
+    it('should have a View As toolbar', () => {
+        const viewAsstoolbarElement: DebugElement = fixture.debugElement.query(
+            By.css('dot-edit-content-view-as-toolbar')
+        );
+        expect(viewAsstoolbarElement).not.toBeNull();
+    });
+
+    it('should change the page wrapper dimensions', () => {
```

Comment:
This test is not testing the binding of the event on the view as toolbar, it means that if we remove the ```(change...)``` event from the HTML this test will work but the real thing will break.